apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -60,9 +117,23 @@ export class GeneratePreviewUsecase {
     });
   }
 
+  private normalizeExampleVariables(
+    validatedContent: ValidatedContentResponse,
+    origin?: WorkflowOriginEnum,
+    payloadSchema?: JSONSchemaDto
+  ) {
+    if (origin === WorkflowOriginEnum.EXTERNAL && payloadSchema) {
+      const payloadVariableExample = { payload: this.generatePayloadVariableExample(payloadSchema) };
+
+      return mergeObjects(payloadVariableExample, validatedContent.finalPayload as Record<string, unknown>);
```

Comment:
Why is this step necessary? What's the `validatedContent` for?

Reply:
validatedContent is the response of PrepareAndValidateContentUsecase returning (ValidatedContentResponse), it had the data about the example payload(finalPayload) and issue.